core/conversion/converters/impl/element_wise.cpp

```diff
@@ -211,6 +266,37 @@ auto element_wise_registrations TRTORCH_UNUSED =
                     div->setName(util::node_info(n).c_str());
                     auto out = ctx->AssociateValueAndTensor(n->outputs()[0], div->getOutput(0));
 
+                    LOG_DEBUG("Output tensor shape: " << out->getDimensions());
+                    return true;
+                  }})
+        .pattern({"aten::true_divide.Tensor(Tensor self, Tensor other) -> Tensor",
```

Comment:
Why do we need a converter for true divide if it is an alias for `torch.div`? I think it would be better to lower to `aten::div`

Reply:
Is there any test case demo for testing lowering, I not sure how to test such case

Reply:
I can write an example in the next couple days on how I see us doing it